website/templates/emails/new_public_project.html.mako

```diff
@@ -0,0 +1,33 @@
+## -*- coding: utf-8 -*-
+<%inherit file="notify_base.mako"/>
+<%def name="content()">
+    <div style="margin: 40px;">
+        <br>
+        <br>
+        Hello ${fullname},
+        <br>
+        Congratulations on making a public project on the Open Science Framework (OSF)! Now that your project “${project_title}" is public, you’ll be able to take advantage of more OSF features:
+        <ul>
+            <li><a href="http://osf.io/${nid}/files/">The number of downloads of your files will be automatically logged for you</a></li>
```

Comment:
Instead of hard-coding these urls, can you construct them using url_for helpers and pass them to the mako template?